Yarn dev dependency not listed

### Description

The build process requires "yarn" package, but it's not listed as a dependency.
"package.json" should be updated to include this devDependency.

### Steps to reproduce

```bash
$ npm install
$ npm test

> lint-staged@0.0.0-development pretest /path/to/lint-staged
> yarn lint

sh: yarn: command not found
npm ERR! Test failed.  See above for more details.

```

--- a/.lintstagedrc.json
+++ b/.lintstagedrc.json
@@ -1,7 +1,7 @@
 {
   "linters": {
     "*.{js,json,md}": ["prettier --write", "git add"],
-    "*.js": ["yarn lint:base --fix", "git add"],
+    "*.js": ["npm run lint:base --fix", "git add"],
     ".*{rc, json}": "jsonlint"
   }
 }

--- a/package.json
+++ b/package.json
@@ -18,9 +18,9 @@
     "precommit": "node index.js",
     "cz": "git-cz",
     "lint:base": "eslint --rule \"prettier/prettier: 2\"",
-    "lint": "yarn lint:base -- .",
-    "lint:fix": "yarn lint --fix",
-    "pretest": "yarn lint",
+    "lint": "npm run lint:base -- .",
+    "lint:fix": "npm run lint --fix",
+    "pretest": "npm run lint",
     "test": "jest --coverage",
     "test:watch": "jest --watch"
   },

